packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -469,12 +469,12 @@ export interface DatabaseInstanceNewProps {
  * A new database instance.
  */
 abstract class DatabaseInstanceNew extends DatabaseInstanceBase implements IDatabaseInstance {
-  public readonly securityGroupId: string;
   public readonly vpc: ec2.IVpc;
+  public readonly connections: ec2.Connections;
 
   protected readonly vpcPlacement?: ec2.SubnetSelection;
   protected readonly newCfnProps: CfnDBInstanceProps;
-  protected readonly securityGroup: ec2.SecurityGroup;
+  protected readonly securityGroups: ec2.ISecurityGroup[];
```

Comment:
So, now that you moved creating `connections` here - is there any need to have this field? I believe the field can now be removed, and `securityGroups` can be just a local variable of the constructor of `DatabaseInstanceNew`, correct?

Reply:
Correct!